res/controllers/Keyboard-Default.kbd.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- TODO(Tomasito): Change this to the version in which KeyboardController support will be added-->
+<MixxxKeyboardPreset schemaVersion="1" mixxxVersion="2.0.0+">
+  <info>
+    <name>Keyboard Default</name>
+    <author>Mixxx</author>
+    <description>Default keyboard mapping. (Layout: en_US)</description>
+    <wiki>https://www.mixxx.org/manual/latest/chapters/appendix.html#appendix-keyboard</wiki>
+  </info>
+
+  <controller>
+    <group name="[AutoDJ]">
```

Comment:
XML format:
IMHO the size of the file is not that important compared to maintaining two XML parsers. 
But If you need a separate parser anyway, this argument is void.